cartography/intel/aws/rds.py

```diff
@@ -216,6 +216,114 @@ def load_rds_instances(
+    neo4j_session: neo4j.Session, data: Dict, region: str, current_aws_account_id: str,
+    aws_update_tag: int,
+) -> None:
+    """
+    Ingest the RDS snapshots to neo4j and link them to necessary nodes.
+    """
+    ingest_rds_snapshot = """
+    UNWIND $Snapshots as rds_snapshot
+        MERGE (snapshot:RDSSnapshot{id: rds_snapshot.DBSnapshotArn})
+        ON CREATE SET snapshot.firstseen = timestamp(),
+            snapshot.arn = rds_snapshot.DBSnapshotArn
+        SET snapshot.db_snapshot_identifier = rds_snapshot.DBSnapshotIdentifier,
+            snapshot.db_instance_identifier = rds_snapshot.DBInstanceIdentifier,
+            snapshot.snapshot_create_time = rds_snapshot.SnapshotCreateTime,
+            snapshot.engine = rds_snapshot.Engine,
+            snapshot.allocated_storage = rds_snapshot.AllocatedStorage,
+            snapshot.status = rds_snapshot.Status,
+            snapshot.port = rds_snapshot.Port,
+            snapshot.availability_zone = rds_snapshot.AvailabilityZone,
+            snapshot.vpc_id = rds_snapshot.VpcId,
+            snapshot.instance_create_time = rds_snapshot.InstanceCreateTime,
+            snapshot.master_username = rds_snapshot.MasterUsername,
+            snapshot.engine_version = rds_snapshot.EngineVersion,
+            snapshot.license_model = rds_snapshot.LicenseModel,
+            snapshot.snapshot_type = rds_snapshot.SnapshotType,
+            snapshot.iops = rds_snapshot.Iops,
+            snapshot.option_group_name = rds_snapshot.OptionGroupName,
+            snapshot.percent_progress = rds_snapshot.PercentProgress,
+            snapshot.source_region = rds_snapshot.SourceRegion,
+            snapshot.source_db_snapshot_identifier = rds_snapshot.SourceDBSnapshotIdentifier,
+            snapshot.storage_type = rds_snapshot.StorageType,
+            snapshot.tde_credential_arn = rds_snapshot.TdeCredentialArn,
+            snapshot.encrypted = rds_snapshot.Encrypted,
+            snapshot.kms_key_id = rds_snapshot.KmsKeyId,
+            snapshot.timezone = rds_snapshot.Timezone,
+            snapshot.iam_database_authentication_enabled = rds_snapshot.IAMDatabaseAuthenticationEnabled,
+            snapshot.processor_features = rds_snapshot.ProcessorFeatures,
+            snapshot.dbi_resource_id = rds_snapshot.DbiResourceId,
+            snapshot.original_snapshot_create_time = rds_snapshot.OriginalSnapshotCreateTime,
+            snapshot.snapshot_database_time = rds_snapshot.SnapshotDatabaseTime,
+            snapshot.snapshot_target = rds_snapshot.SnapshotTarget,
+            snapshot.storage_throughput = rds_snapshot.StorageThroughput,
+            snapshot.region = $Region,
+            snapshot.lastupdated = $aws_update_tag
+        WITH snapshot
+        MATCH (aa:AWSAccount{id: $AWS_ACCOUNT_ID})
+        MERGE (aa)-[r:RESOURCE]->(snapshot)
+        ON CREATE SET r.firstseen = timestamp()
+        SET r.lastupdated = $aws_update_tag
+    """
+    snapshots = []
+
+    for snapshot in data:
```

Comment:
Put this functionality into a separate function you can call `transform_rds_snapshots`